README.md

```diff
@@ -1 +1,75 @@
-# to-do-list-
\ No newline at end of file
+# Todo List App
+
+This is a simple todo list app to help you plan and manage your time effectively, by writing down all the tasks that you have in a given day.
+
+## Built With
+
+- Major languages: HTML & Javascript
+- Preprocessor: Sass
+- Technologies used: Linters, Git and Github, webpack
+
+## Live Demo
+
+N/A
+
+## Getting Started
+
+### Prerequisites
+
+You should be having [Node.js](https://nodejs.org/en/) installed on your computer
+
+### Setup
+
+To get a local copy up and running follow these simple example steps.
+
+1. Navigate to a folder where you want the cloned file to appear
+
+2. Clone this repo in your terminal or git bash using the command
+
+   ``` git@github.com:John-Kibirige/to-do-list.git ```
+
+3. run `npm install ` to install all the necessary dependencies including the linters and webpack
+
+4. Navigate to the folder called to-do-list and open this project using your editor
+
+5 You run `npm run start` to start the local server on port 3000
+
+6 You also run `npm run build` to build your project
+
+### Run tests (For the linters added)
+
+These were linter tests mainly
+
+- For css, run `npx hint .`
+- Html, run `npx stylelint "**/*.{css,scss}"`
+- Javascript `npx eslint .`
+
+### Deployment
+
+## Author
+
+👤 **Kibirige John**
+
+- Github: [John-Kibrige](https://github.com/John-Kibirige)
+- Twitter: [kibirigejohn005](https://twitter.com/kibirigejohn005)
+- LinkedIn: [Kibirige John](https://www.linkedin.com/in/kibirige-john-64160520a/trk=public_profile_samename-profile_profile-result-card_result-card_full-click&original_referer=https%3A%2F%2Fwww%2Egoogle%2Ecom%2F&originalSubdomain=ug)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```